README.md

```diff
@@ -40,7 +40,7 @@
 ---
 
 ## Overview
-[Rotki](https://rotki.com/) is an open source portfolio tracking, analytics, accounting, and tax reporting tool that protects your privacy.  The mission of rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly, unlike virtually every other competing service which consists of closed source SaaS onto which you are forced to hand over all your financial data, with rotki your data is stored encrypted locally in your computer. It enables you to take ownership of your financial data!
+[Rotki](https://rotki.com/) is an open source portfolio tracking, analytics, accounting, and tax reporting tool that protects your privacy.  The mission of Rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly, unlike virtually every other competing service which consists of a closed source SaaS onto which you are forced to hand over all your financial data, with rotki your data is stored encrypted locally on your computer. It enables you to take ownership of your financial data!
```

Comment:
rotki is consistently kept as lowercase everywhere. Nothing wrong here. If you want to consider anything just keep the in->on change and discard everything else.

**DO NOT** make a new PR for this. Just do it here by force pushing into the same PR.

Reply:
I have made the necessary changes